packages/expo-av/android/src/main/java/expo/modules/av/AVManager.java

```diff
@@ -89,6 +90,7 @@
   private long mAudioRecorderDurationAlreadyRecorded = 0L;
   private boolean mAudioRecorderIsRecording = false;
   private boolean mAudioRecorderIsPaused = false;
+  private boolean isRegistered = false;
```

Comment:
Android Expo codebase tends to follow AOSP code guidelines, so I would be grateful if you would rename this variable to `mIsRegistered`. 🙂

Reply:
Fixed.